docs/maintenance/Versioning.mdx

```diff
@@ -15,11 +15,21 @@ Additionally, we promote to the `latest` tag on NPM once per week, **on Mondays
 
 The latest version under the `latest` tag is:
 
-<a href="https://www.npmjs.com/package/@typescript-eslint/parser"><img src="https://img.shields.io/npm/v/@typescript-eslint/parser/latest.svg?style=flat-square" alt="NPM Version" /></a>
+<a href="https://www.npmjs.com/package/@typescript-eslint/parser">
+  <img
+    src="https://img.shields.io/npm/v/@typescript-eslint/parser/latest.svg?style=flat-square"
+    alt="NPM Version"
+  />
+</a>
 
 The latest version under the `canary` tag **(latest commit to `main`)** is:
 
-<a href="https://www.npmjs.com/package/@typescript-eslint/parser"><img src="https://img.shields.io/npm/v/@typescript-eslint/parser/canary.svg?style=flat-square" alt="NPM Version" /></a>
+<a href="https://www.npmjs.com/package/@typescript-eslint/parser">
+  <img
+    src="https://img.shields.io/npm/v/@typescript-eslint/parser/canary.svg?style=flat-square"
+    alt="NPM Version"
+  />
+</a>
```

Comment:
🤷 I guess Prettier more aggressively formats inline HTML snippets in `.mdx` files than `.md`.